docs/en/index.md

```diff
@@ -2,21 +2,57 @@
 title: Home
 ---
 
-## A simple extensible CSS framework, written in Sass.
+## Get started
 
-[![Build Status](https://travis-ci.org/vanilla-framework/vanilla-framework.svg?branch=master)](https://travis-ci.org/vanilla-framework/vanilla-framework)
-[![npm version](https://badge.fury.io/js/vanilla-framework.svg)](http://badge.fury.io/js/vanilla-framework)
-[![Downloads](https://img.shields.io/npm/dm/vanilla-framework.svg)](https://www.npmjs.com/package/vanilla-framework)
-[![devDependency Status](https://david-dm.org/vanilla-framework/vanilla-framework/dev-status.svg)](https://david-dm.org/vanilla-framework/vanilla-framework#info=devDependencies)
-[![Chat in #vanilla-framework on Freenode](https://img.shields.io/badge/chat-%23vanilla--framework-blue.svg)](http://webchat.freenode.net/?channels=vanilla-framework)
+You can use Vanilla in your projects in a few different ways.
 
-Vanilla Framework contains a basic CSS grid and pattern classes, and is designed to be extended either directly or by creating extension themes.
+### Install via npm
 
-Sites created within Canonical should follow this style guide closely, whereas external sites are free to adapt and expand the existing components to their needs.
+The recommended way to get Vanilla is through npm:
 
-For ways to use Vanilla framework, see [vanilla-framework/vanilla-framework on GitHub](https://github.com/vanilla-framework/vanilla-framework).
+`npm install --save vanilla-framework`
```

Comment:
I don't remember concurring. The difference between `--save` and `--save-dev` is that in the former case the dependency will be included in all *child project*. That is, if `my-project` `--save`s `vanilla-framework`, and then `ants-project` adds `my-project` as a dependency, `ants-project` will also get `vanilla-framework`.

I don't think this is at all relevant for the expected use-case. I think in normal websites, `vanilla-framework` is simply a build resource, only relevant for local building, not an actual dependency for the website project itself, which actually just needs some built CSS.

So I think we should use `--save-dev`.

I realise this has now been merged, but I think I'll file an issue about it now.